Mac/AppDelegate.m

```diff
@@ -1,23 +1,64 @@
-#import "AppDelegate.h"
 #import "CCPackageManager.h"
 #import "CCDirector_Private.h"
-#import "AppController.h"
+#import "TestbedSetup.h"
+#import "MainMenu.h"
 
-@interface AppDelegate ()
-
-@property (weak) IBOutlet NSWindow *window;
-@property (weak) IBOutlet CC_VIEW<CCView> *glView;
+@interface AppDelegate : NSObject <NSApplicationDelegate, NSWindowDelegate>
 @end
 
 @implementation AppDelegate
+{
+    NSWindow *_window;
+    CC_VIEW<CCView> *_view;
+}
 
 - (void)applicationDidFinishLaunching:(NSNotification *)aNotification
 {
-    AppController *controller = [AppController sharedController];
-    controller.window = _window;
-    controller.glView = _glView;
-    [controller setupApplication];
-    return;
+#if 0
```

Comment:
Yeah. There are two ways to configure a cocos2d application. In your template code, you can call setupApplication, or you can do the setup manually.

`setupApplication` has a bunch of defaults for setting up a project with the normal spritebuilder settings. If a person wants to override that behavior, they can provide their own code - setting special search paths, doign something funky with the window, other content scale settings... basically anything that involves `[CCSetup createCustomSetup];` instead of `setupApplication`.

Both implementations are provided here so we can test both code paths by flipping the ifdef. Maybe not ideal, but better than having two targets, or something like that.